FirebaseRemoteConfigSwift/Exporter/Exporter.swift

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO(ncooke3): Add warning that importing this extension is deprecated.
```

Comment:
We could either initially or eventually add a warning here before the eventual removal of the target. I think I'm leaning towards a conservative approach the warning is added later.

Reply:
Agreed